docs/map-building/tiled-editor/publish/wa-hosted.md

```diff
@@ -67,8 +67,8 @@ npm run upload -- -k your-api-key -u your-mapstorage-url -d your-directory
    There you can create a new token. (Don't forget to save it !)
 
 3. **Directory:**
-   You can also add a `directory name` if you want. It will be the folder where all your uploaded files will be stored in.
-   If you leave this blank, there will be no directory.
+   You also have to add a `directory name`, it will be the folder where all your uploaded files will be stored in.
+   If you want ot store your map in a specific directory, you can edit this in the .env file or if your don't put one, by default it will be your github name and github repository like this : `githubname-githubrepository`. The directory name appear in the URL of your map. This is done to avoid errors and push two maps with the same folder onto Workadventure servers.
```

Comment:
Ok, I failed to see that :)

The default directory name cannot be `githubname-githubrepository`. Because there is absolutely no guarantee that:

1. The user cloned the repository (he could download the ZIP file of the starter kit). In the case, there is no Git repository (the very point of the map uploader is to be able to not use Git)
2. The user might not even have Git installed, (in which case this code will fail: https://github.com/workadventure/upload-maps/blob/master/src/upload.ts#L85-L104)

You need to hard code a directory name instead. Just put `maps`.